packages/jest-runner/src/run_test.js

```diff
@@ -98,15 +99,24 @@ async function runTestInternal(
     testConsole = new BufferedConsole();
   }
 
-  const cacheFS = {[path]: testSource};
+  let cacheFS = {[path]: testSource};
   setGlobal(environment.global, 'console', testConsole);
 
-  const runtime = new Runtime(config, environment, resolver, cacheFS, {
+  const coverageOptions = {
     collectCoverage: globalConfig.collectCoverage,
     collectCoverageFrom: globalConfig.collectCoverageFrom,
     collectCoverageOnlyFrom: globalConfig.collectCoverageOnlyFrom,
     mapCoverage: globalConfig.mapCoverage,
-  });
+  };
+
+  let runtime = new Runtime(
+    config,
```

Comment:
I think it's time to turn this constructor into something that takes an object.

Reply:
Should we make this change now? This would break `Runtime`'s interface; that's why I thought it was better to keep it as-is.

Reply:
No, that should be a follow-up but we could do it for Jest 22.